src/AppDevRouter.js

```diff
@@ -10,104 +12,106 @@ export type RequestType = 'GET' | 'POST' | 'DELETE';
 
 /**
  * AppDevResponse - the response from an HTTP request
- * 
+ *
  * Wraps a `success` field around the response data
  */
 class AppDevResponse<T> {
   success: boolean;
+
```

Comment:
why the empty line?

Reply:
my linter added it saying `Expected new line between class members`

Reply:
fake news

Reply:
<img width="643" alt="screen shot 2019-02-14 at 9 20 29 pm" src="https://user-images.githubusercontent.com/26048121/52830333-60e38400-309e-11e9-9959-16e78163f503.png">

Reply:
i'm with @cswenberg i don't think this is standard, can you add to `.eslintrc` to not check this rule?